src/common/src/catalog/column.rs

```diff
@@ -128,27 +126,14 @@ pub struct ColumnDesc {
 
 impl ColumnDesc {
     pub fn unnamed(column_id: ColumnId, data_type: DataType) -> ColumnDesc {
```

Comment:
Maybe we could rename it to `unnamed_for_test`

Reply:
Or just `for_test`

Reply:
I was confused when we have `unnamed` column, and found it's just in tests.

Reply:
Counting on @xiangjinwu! 😁